roles/ceph-osd/tasks/bcache.yml

```diff
@@ -47,6 +47,10 @@
   with_items: ceph.disks
   when: ok_to_deploy
 
+- name: udevadm trigger
```

Comment:
Yeah.  The bcache devices were not showing up and the play was timing out.  It also didn't recover very well on subsequent runs since it assumes that if the partitions have been created all the tasks were completed successfully.

It seems that the make-bcache command is not triggering udev to create the device nodes like it should.  Running "udevadm trigger" after the make-bcache commands execute ensures that they are created before proceeding.